bears/elm/ElmLintBear.py

```diff
@@ -0,0 +1,37 @@
+from shutil import which
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='elm-format',
+        use_stdin=True,
```

Comment:
stdin should be avoided if possible.  The filename cant be specified on the command line?